src/styles/blocks/card.scss

```
@@ -0,0 +1,140 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-grow: 0;
+  flex-shrink: 0;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 32px 16px 16px;
+  flex-wrap: wrap;
+  height: $card-height;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:nth-child(5),
+  &:nth-child(1) {
+    margin-left: 3px;
+  }
+
+  &:not(:nth-child(4n)) {
+    margin-right: 48px;
+    margin-bottom: 46px;
+  }
+
+  &__image-imac {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 40px;
+  }
+
+  &__image {
+    display: flex;
+    justify-content: center;
+    @include iMacSize($imac-height, $imac-width);
+  }
+
+  &__name {
+    font-weight: 500px;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-text-color;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    align-self: flex-start;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    padding-bottom: 17px;
+    color: $secondary-text-color;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    height: 16px;
+    padding-bottom: 23px;
+  }
+
+  &__stars {
+    display: flex;
+    justify-content: space-between;
+    line-height: 16px;
+    margin-bottom: 23px;
+
+  }
+
+  &__stars .star-1,
+  &__stars .star-2,
+  &__stars .star-3,
+  &__stars .star-4 {
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    background: url("../images/star-active.svg");
+    background-repeat: no-repeat;
+    background-position: top;
+  }
+
+  &__stars .star-5 {
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    background: url("../images/star.svg");
+    background-repeat: no-repeat;
+    background-position: top;
+  }
```

```suggestion
  &__stars {
    display: flex;
    justify-content: space-between;
    line-height: 16px;
    margin-bottom: 23px;
    height: 16px;
    width: 16px;
    margin-right: 4px;
    background-repeat: no-repeat;
    background-position: top;
    background: url("../images/star.svg");
  }

  &__stars .star-1,
  &__stars .star-2,
  &__stars .star-3,
  &__stars .star-4 {
    background: url("../images/star-active.svg");
  }

  }
```

так уникаєш повторення коду, а також спробуй for loop використати.